core/src/main/java/org/mule/runtime/core/internal/policy/CompositeOperationPolicy.java

```diff
@@ -130,4 +137,11 @@ public CoreEvent process(CoreEvent event) throws MuleException {
       return error(e);
     }
   }
+
+  private void logOperationEvent(EventContext eventContext, Message message) {
+    if (LOGGER.isTraceEnabled()) {
+      LOGGER.trace("Executing operation\n" + eventContext.toString() + "\n"
```

Comment:
do you need to add the context id? it should be included as part of the context information logged in every line. Same applies in other parts of this PR.